src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -43,6 +44,7 @@ import reducer, { init } from './reducer';
  * Component that renders the incident filter form
  */
 const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, onUpdateFilter }) => {
+  const { sources } = useContext(AppContext);
```

Comment:
What if you provide the value for `sources` to the `IncidentManagementContext.Provider` component? That way you'd only have to test a single provider for the `FilterForm` component.

Reply:
True. I was thinking of setting the `sources` prop in the `IncidentManagementContext.Provider`, so that the `incident-management` module only uses the `IncidentManagementContext` instead of using both `IncidentManagementContext` and `AppContext`, but there's no real benefit in doing that, I guess.